docs/source/workflows/predictors.rst

```diff
@@ -397,3 +397,93 @@ Data Sources
+Here we show an example of how to combine these modules to accomplish those many suitable tasks.
+
+Using ExpressionPredictors to perform pre-processing can be used to featurize data, which is a valuable way to leverage domain knowledge by transforming raw inputs into quantities known to be relevant.
+In the example below, we use an ExpressionPredictor to annotate the training data with a "hydration ratio".
+The hydration ratio is the mass ratio of water to flour.
+Bakers know this quantity to be of fundamental importance to the taste and texture of bread, so computing this quantity might be expected to help the SimpleMLPredictor make more efficient use of scarce training data.
+(In a more standard materials science context, an ExpressionPredictor might be used to annotate semiconductor data with an analytical expression of idealized electron mobility as a function of dopant concentrations.)
+
+In the example below, we use the ExpressionPredictor feature to compute a bread loaf product's shelf life.
+This simulates a scenario where shelf life is determined by a quality control rule of a few physically measurable quantities: ``final pH`` and ``final hydration`` as estimated by the SimpleMLPredictor, as well as the fraction of salt in the ingredients.
+Using ExpressionPredictors in this manner to post-process learned data is often useful for displaying information on the platform based on transformations of the learned physical properties.
+This pattern is also extremely useful for performing optimization over complex objectives: in the following example, we can use shelf life as an objective or constraint in a :doc:`DesignWorkflows <design_workflows>`.
+
+.. code:: python
+
+    from citrine.informatics.descriptors import RealDescriptor
+    from citrine.informatics.predictors import ExpressionPredictor, GraphPredictor, SimpleMLPredictor
+
+    ######## Omitted step: create DataSource with columns associated with the following descriptors ########
+    # wheat_flour_quantity = RealDescriptor('wheat flour mass', lower_bound=300, upper_bound=550, units="g")
+    # rye_flour_quantity = RealDescriptor('rye wheat flour mass', lower_bound=0, upper_bound=100, units="g")
+    # water_quantity = RealDescriptor('water mass', lower_bound=200, upper_bound=400, units="g")
+    # salt_quantity = RealDescriptor('salt mass', lower_bound=4, upper_bound=8, units="g")
+    # starter_quantity = RealDescriptor('starter mass', lower_bound=5, upper_bound=30, units="g")
+    # final_ph = RealDescriptor('final pH', lower_bound=2.5, upper_bound=5, units="")
+    # final_loaf_hydration = RealDescriptor('final loaf hydration', lower_bound=0, upper_bound=100, units="")
+    #
+    # data_source = create_ara_data_source_from_breads_gemd(descriptors=[...descriptors above...])
+
+    dough_hydration = RealDescriptor('dough hydration', lower_bound=0, upper_bound=1)
+    shelf_life = RealDescriptor('shelf life', lower_bound=0, upper_bound=72)
```

Comment:
```suggestion
    shelf_life = RealDescriptor('estimated shelf life', lower_bound=0, upper_bound=72)
```
I'm not sure about this suggestion.  Is the purpose of this expression to actually predict the shelf life (such that you'd want to compare it to true shelf-life values) or is it to construct a shelf life figure of merit or indicator that we want to maximize?

Reply:
Idea is to construct a figure of merit -- in the stated motivation, it's a business rule based on physical properties, so there would be no true values.

Reply:
If there would be no true values, I'd try to make that clearer via the name.  "estimated shelf life" or "shelf life metric" might be better, or something else.  Could we use the term "figure of merit" in the text of the example as well?

Reply:
How is `approximate shelf life`?